go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/jinzhu/gorm v1.9.16
 	github.com/justinas/alice v1.2.0
 	github.com/kr/text v0.2.0 // indirect
+	github.com/lib/pq v1.3.0
```

Comment:
remove manually and then run `go mod tidy`

Reply:
Done.